src/opm/output/eclipse/EclipseIO.cpp

```diff
@@ -160,7 +159,7 @@ bool EclipseIO::Impl::wantRFTOutput( const int  report_step,
 {
     return !isSubstep
         && (static_cast<std::size_t>(report_step)
-            >= this->schedule.rftConfig().firstRFTOutput());
+            >= this->schedule.first_RFT());
```

Comment:
I'm probably missing something, but
```C++
report_step >= std::optional<std::size_t>{}
```
is true for all values of `report_step` so this predicate is equivalent to
```C
return !isSubStep;
```
if `first_RFT()` returns a default-constructed `optional<std::size_t>` (i.e., a `std::nullopt`).  Is that really the intended behaviour of `first_RFT()`?  Does this get us back to the state where we always create empty `.RFT` files if the case does **not** request RFT output?

Reply:
Ehh - no; this was a bug - fix here: https://github.com/OPM/opm-common/pull/2308